CONTRIBUTING.md

```diff
@@ -26,6 +27,14 @@ We also maintain a [gitter chat room][link_gitter] for more informal conversatio
 There is significant cross-talk between these two spaces, and we look forward to hearing from you in either venue!
 As a reminder, we expect all contributions to `tedana` to adhere to our [code of conduct][link_coc].
 
+### Monthly developer calls
+
+We run monthly developer calls via Zoom.
+You can see the schedule via the `tedana` [google calendar](https://calendar.google.com/calendar/embed?src=pl6vb4t9fck3k6mdo2mok53iss%40group.calendar.google.com&ctz=Europe%2FLondon).
+An agenda will be circulated in the gitter channel in advance of the meeting.
```

Comment:
Hm, I feel that if we say one will be circulated, we should say how far in advance. We should probably also decide a few more details on how the agenda will be managed, but that's out of the scope of this PR.

Reply:
Yeaaaaah.... I feel like that would be nice..... but difficult to commit to at the moment...

Reply:
Fair enough; looks great!